pulseapi/users/serializers.py

```diff
@@ -1,7 +1,24 @@
 """Serialize the models"""
 from rest_framework import serializers
 
-from pulseapi.users.models import EmailUser
+from pulseapi.users.models import (
+    EmailUser,
+    UserFavorites,
+)
+
+
+class UserFavoritesSerializer(serializers.ModelSerializer):
+    """
+    Serializes a {user,entry,when} fave.
+    """
+
+    user = '__all__'
```

Comment:
Actually, I don't know what this is :P I thought all serializer fields should be instances. Are you sure this is working?